.drone.yml

```diff
@@ -1,30 +0,0 @@
-image: clever/drone-node:6.2.2
-notify:
-  email:
-    recipients:
-    - drone@clever.com
-  slack:
-    on_failure: true
-    on_started: false
-    on_success: false
-    webhook_url: $$slack_webhook
-publish:
-  report_card: {}
-  catapult:
```

Comment:
Can't merge this until Catapult publish CI script is built out (hopefully by tomorrow)